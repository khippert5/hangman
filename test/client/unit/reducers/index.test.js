import indexReducer, { defaultState } from '../../../../src/client/reducers';

describe('Index verify reducer unit tests', () => {
  const solution = {
    word: 'test',
    hint: 'Test answer',
  }

  it('Match snapshot. This will fail if new state default attributes change', async () => {
    const newState = defaultState;
    newState.ui.solution = solution
    expect(newState).toMatchSnapshot();
  });
});
