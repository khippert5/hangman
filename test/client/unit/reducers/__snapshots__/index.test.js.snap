// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index verify reducer unit tests Match snapshot. This will fail if new state default attributes change 1`] = `
Object {
  "app": Object {
    "init": Object {
      "date": null,
      "time": null,
    },
  },
  "ui": Object {
    "correct": null,
    "error": null,
    "incorrect": 4,
    "selected": Array [],
    "solution": Object {
      "hint": "Test answer",
      "word": "test",
    },
    "success": false,
    "validLetters": Array [],
    "values": Array [
      "a",
      "b",
      "c",
      "d",
      "e",
      "f",
      "g",
      "h",
      "i",
      "j",
      "k",
      "l",
      "m",
      "n",
      "o",
      "p",
      "q",
      "r",
      "s",
      "t",
      "u",
      "v",
      "w",
      "x",
      "y",
      "z",
    ],
  },
}
`;
